frontend/src/app/supportAdmin/SupportHomeLink.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import iconSprite from "../../../node_modules/@uswds/uswds/dist/img/sprite.svg";
+import { LinkWithQuery } from "../commonComponents/LinkWithQuery";
+
+const SupportHomeLink: React.FC<any> = () => {
```

Comment:
ooo thank you for adding this -- will add it to the unarchive patients page in #6064 😸